californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfig.java

```diff
@@ -192,6 +192,8 @@
 		public static final String TCP_CONNECTION_IDLE_TIMEOUT = "TCP_CONNECTION_IDLE_TIMEOUT";
 		public static final String TCP_CONNECT_TIMEOUT = "TCP_CONNECT_TIMEOUT";
 		public static final String TCP_WORKER_THREADS = "TCP_WORKER_THREADS";
+		public static final String BERT_OPTION = "BERT_OPTION";
+		public static final String BERT_OPTION_STEP_SIZE = "BERT_OPTION_STEP_SIZE";
```

Comment:
I agree. We could check the TCP_NUMBER_OF_BULK_BLOCKS value and then decide if BERT is enabled.